### PR TITLE
change lintr URLs to r-lib org

### PR DESCRIPTION
**Pull request by dpprdan**
This replaces `jimhester` with `r-lib` in the GitHub URLs / repo address.

Questions:

1. The URL in the `lint-changed-file.yaml` needs to be changed as well, I assume?
    https://github.com/r-lib/lintr/blob/4ca7e484bcbe538cc5a81338c0ec55a0ae084cb0/.github/workflows/lint-changed-files.yaml#L35
2. Is the [gitter chat](https://github.com/dpprdan/lintr/blob/9fa72489b5d641312ca7c27128599298a7f0b506/README.md?plain=1#L5) still maintained? The last activity was over a year ago, AFAICT.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # lintr
-[![R build status](https://github.com/jimhester/lintr/workflows/R-CMD-check/badge.svg)](https://github.com/jimhester/lintr/actions)
-[![codecov.io](https://codecov.io/github/jimhester/lintr/coverage.svg?branch=master)](https://codecov.io/github/jimhester/lintr?branch=master)
-[![CRAN_Status_Badge](https://www.r-pkg.org/badges/version/lintr)](https://cran.r-project.org/package=lintr) [![Join the chat at https://gitter.im/jimhester-lintr/Lobby](https://badges.gitter.im/jimhester-lintr/Lobby.svg)](https://gitter.im/jimhester-lintr/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![R build status](https://github.com/r-lib/lintr/workflows/R-CMD-check/badge.svg)](https://github.com/jimhester/lintr/actions)
```
Comment (AshesITR) [inline README.md:2]: The link targets also contain /jimhester URLs

**Review by AshesITR — CHANGES_REQUESTED** [review]
Thanks for this PR! Just a few things need adapting.

**Comment by dpprdan** [comment]
> Questions:
> 
> 1. The URL in the `lint-changed-file.yaml` needs to be changed as well, I assume?
>        https://github.com/r-lib/lintr/blob/4ca7e484bcbe538cc5a81338c0ec55a0ae084cb0/.github/workflows/lint-changed-files.yaml#L35
> 2. Is the [gitter chat](https://github.com/dpprdan/lintr/blob/9fa72489b5d641312ca7c27128599298a7f0b506/README.md?plain=1#L5) still maintained? The last activity was over a year ago, AFAICT.


Can you comment on these, @AshesITR?

**Comment by AshesITR** [comment]
I think it should be safe to substitute all occurrences of `jimhester` with `r-lib` for `github.com` URLs.
Regarding codecov, I don't know. @jimhester do you know if the codecov repo needs moving?

I've personally never looked at the gitter channel, not sure about the other maintainers, though.

cc @MichaelChirico

**Review by AshesITR — APPROVED** [review]

**Comment by MichaelChirico** [comment]
also haven't looked at gitter

**Review by MichaelChirico — APPROVED** [review]

**Comment by dpprdan** [comment]
> Regarding codecov, I don't know. @jimhester do you know if the codecov repo needs moving?

Most recent commit on [`jimhester`](https://codecov.io/github/jimhester/lintr?branch=master) is 6 months old, whereas [`r-lib`](https://codecov.io/github/r-lib/lintr?branch=master) a few minutes ago. So I'd say this needs moving.

> also haven't looked at gitter

Sounds unmaintained, so I'll delete the link then?

**Comment by MichaelChirico** [comment]
Merging for now. Jim, feel free to mention any changes we should revert later.

**Pull request merged**